### DISCUSSION TITLE
Can't change session keybindings when changing i3 super key

### DISCUSSION THREAD

**Discussion by captainvera** [regolith-linux/regolith-desktop#678] [Q&A]
Hello, 

First of all, if the right place for this kind of thing is the issues let me know and I'll gladly move there :)

I am on regolith 2.1 / ubuntu jammy. Did a full install from scratch.

Everything has been working fine so far except that after I changed the i3 `super` key as shown in the docs:
```
i3-wm.mod: Mod1
i3-wm.alt: Mod4
```

All of the session keybindings have stopped working (exit app, logout, sleep, shutdown, etc, etc) and I can't figure out how to re-configure them.. 

Should it be through Xresources or through the gnome settings panel? I couldn't find any of the relevant keybinds in the latter. 

Thanks in advance!

**Comment by kgilmer** [comment]
Hi @captainvera , hmm I cannot reproduce the problem you are seeing.  I added the same two lines to my `~/.config/regolith2/Xresources` file and after refreshing i3 all of the session keybindings worked correctly with `alt` as expected.  Did you have any other customizations present by chance?

**Reply by captainvera** [reply]
Hey @kgilmer Sorry it took me a while to get back to you. My `~/.config/regolith2/Xresources` file is the following:

```
!regolith.look.path: /usr/share/regolith-look/nevil
regolith.look.path: /usr/share/regolith-look/nord

i3-wm.mod: Mod1
i3-wm.alt: Mod4
i3-wm.gaps.inner.size: 15

i3-wm.binding.file_search: Mod4+space
i3-wm.binding.ws_next2: Mod4+Right
i3-wm.binding.ws_prev2: Mod4+Left
i3-wm.binding.take_next_free: Mod4+grave

```

I don't see how any of it could interfere with the session keybindings. Other than that I changed the caps to esc using `gnome-tweaks` but I don't see how that could have an impact.

I am a bit lost on how I could debug this further. All of the keybindings work perfectly after the changes _except_ the session keybindings.

**Reply by kgilmer** [reply]
As a sanity check, can you temporarily comment out everything below `i3-wm.alt: Mod4` and log back in to see if the behavior varies?

**Reply by kgilmer** [reply]
when you login do you see any errors related to i3 in `/var/log/syslog`?

**Reply by captainvera** [reply]
This is what I see on the `syslog` related to i3:

```
Aug  5 12:14:35 fastbook regolith.desktop[2372]: <3>[rofication] Command '/usr/share/i3xrocks/scripts/$BLOCK_NAME' exited unexpectedly with code 1
Aug  5 12:15:38 fastbook regolith.desktop[2362]: [../i3bar/src/xcb.c:863] ERROR: Error getting _XEMBED_INFO property: error_code 3
```

Doesn't seem related...

**Reply by captainvera** [reply]
And unfortunately commenting everything out changed nothing... Where are these system keybindings defined and how could I change them if I was going to overwrite them manually?

**Reply by kgilmer** [reply]
Hmm, weird.  Unsure of why it's not working for you at the moment.  All Regolith i3 packages that contribute config partials are specified here: https://regolith-desktop.com/docs/using-regolith/configuration/#all-i3-configuration-packages.  Another way of discovering this is by searching in the partial directory (all default partials live in a single directory), for example `find /usr/share/regolith/i3/config.d/ | xargs grep -HsF Session`.  The one in particular you want is `/usr/share/regolith/i3/config.d/55_session_keybindings`.

See here for details of how to "lift and shift" this config partial into your user config: https://regolith-desktop.com/docs/howtos/customize-i3-configuration/#how-to-customize-a-default-partial
